routes/web.php

```diff
@@ -13,12 +16,30 @@
 |
 */
 
-Route::get('/', function () {
-    $owners = \App\Http\Resources\OwnerResource::collection(\App\Models\Owner::all());
-    return view('welcome')->with('owners', $owners);
+Route::get('/', function (OwnerRepositoryInterface $ownerRepository) {
+    return view('welcome')->with('owners', $ownerRepository->getAll());
 });
 
+
 Route::resources([
-    'owner.cars' => \App\Http\Controllers\CarController::class,
-    'owner' => \App\Http\Controllers\OwnerController::class,
+    'owner.cars' => CarController::class,
 ]);
+
+Route::get('/owner', [OwnerController::class, 'index'])->name('owner.index');
+Route::post('/owner', [OwnerController::class, 'store'])->name('owner.store');
+Route::get('/owner/create', [OwnerController::class, 'create'])->name('owner.create');
+Route::get('/owner/{ownerId}/edit', [OwnerController::class, 'edit'])->name('owner.edit');
+Route::get('/owner/{ownerId}', [OwnerController::class, 'show'])->name('owner.show');
+Route::put('/owner/{ownerId}', [OwnerController::class, 'update'])->name('owner.update');
+Route::delete('/owner/{ownerId}', [OwnerController::class, 'destroy'])->name('owner.destroy');
+
+//Route::group(['prefix' => '/owner/{ownerId}', 'as' => 'owners.cars.'], static function () {
+//    Route::get('/cars', [CarController::class, 'index'])->name('index');
+//    Route::post('/cars', [CarController::class, 'store'])->name('store');
+//    Route::get('/cars/create', [CarController::class, 'create'])->name('create');
+//    Route::get('/cars/{carId}/edit', [CarController::class, 'edit'])->name('edit');
+//    Route::get('/cars/{carId}', [CarController::class, 'show'])->name('show');
+//    Route::put('/cars/{carId}', [CarController::class, 'update'])->name('update');
+//    Route::delete('/cars/{carId}', [CarController::class, 'delete'])->name('delete');
+//
+//});
```

Comment:
These are the changes I'd have made if I were to continue with the CarController, this is so we can use the Repository Pattern and not use Route Model Binding